Automattic/woocommerce-payments
# Fix Teams for WooCommerce Memberships on product WooPay Express Checkout Button

#### Changes proposed in this Pull Request








Fix Teams for WooCommerce Memberships fields data not send when clicking WooPay Express Checkout Button on product page.

#### Testing instructions





* Install WooCommerce Memberships and Teams for WooCommerce Memberships plugins.
* Create a team.
* Create a product to join this team.
* On product page, type the team name and check the `Take up a seat` checkbox.
* Click the WooPay button.
* You should be redirected to WooPay with the correct cart data.
* On product page, do not fill out the team name field.
* Click the WooPay button.
* An error message should be shown.

-------------------


**Post merge**
